marlowe-chain-sync/deploy/fix-collateral-index.sql

```diff
@@ -0,0 +1,16 @@
+-- Deploy chain:fix-collateral-index to pg
+-- requires: split-address
+
+BEGIN;
+
+-- Fix for `preview` testnet.
```

Comment:
@jhbertra, Is this too parochial a migration?

Reply:
No, I think it's fine.